realtime/server.js

```diff
@@ -26,7 +26,8 @@ var messages_typing = {};
 function getOptions(form, path, method) {
     var headers = {
         'User-Agent':       'node-ting/0.1.0',
-        'Content-Type':     'application/x-www-form-urlencoded'
+        'Content-Type':     'application/x-www-form-urlencoded',
+        'password':         PASS
```

Comment:
Use the [standard authorization headers](https://en.wikipedia.org/wiki/Basic_access_authentication#Client_side)